libs/ledger-live-common/src/families/avalanchepchain/js-estimateMaxSpendable.ts

```diff
@@ -0,0 +1,21 @@
+import { BigNumber } from "bignumber.js";
+import type { AccountLike, Account } from "@ledgerhq/types-live";
+import { getMainAccount } from "../../account";
+
+/**
+ * Returns the maximum possible amount for transaction
+ *
+ * @param {Object} param - the account, parentAccount and transaction
+ */
+const estimateMaxSpendable = async ({
+  account,
+  parentAccount,
+}: {
+  account: AccountLike;
+  parentAccount?: Account;
+}): Promise<BigNumber> => {
+  const a = getMainAccount(account, parentAccount);
+  return a.spendableBalance;
```

Comment:
Usually we do use js-getStatusTransaction to get the max amount minus fee cost.
I was wondering if there's no fees in PChain ?

Reply:
Correct, there are no fees to delegate on the p chain!